{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/jinja2.py

```diff
@@ -1,6 +1,31 @@
 import jinja2
 from django.templatetags.static import static
 from django.urls import reverse
+from markdown_it import MarkdownIt
+
+markdown_converter = MarkdownIt()
```

Comment:
Change to `MarkdownIt("js-default")` to make it safe.
https://markdown-it-py.readthedocs.io/en/latest/security.html for background